pages/showcase.js

```diff
@@ -85,6 +85,8 @@ const Screen = styled.div`
 `;
 
 const Showcase = ({ router }) => {
+  console.log('HELLO', router.query);
```

Comment:
Probably remove that before merging :wink:

Reply:
Well... this is embarassing 😳

Reply:
Haha no worries, happens to me all the time!